template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/.template.config/template.json

```diff
@@ -73,13 +74,13 @@
         {
           "condition": "(!OrganizationalAuth)",
           "exclude": [
-            "AzureAd/**"
```

Comment:
We would like to get rid of one level of folder nesting. Instead of changing this one back, can we fix the other one to have top level AzureAd and Azure AdB2C folders?